File: Estrutura_Repetição/C++/Ex1114/Main.cpp
#include <iostream>

int main () {

    int senha = 2002;

    while (1) {
        int nova_senha;
        std::cin >> nova_senha;

        if (senha == nova_senha) {
            std::cout << "Acesso Permitido" << std::endl;
            break;
        } else {
            std::cout << "Senha Invalida" << std::endl;
        }
    }

    return 0;
}

File: Estrutura_Repetição/C#/Ex1114/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        int senha = 2002;

        while (true) {
            int novaSenha = int.Parse(Console.ReadLine());

            if (senha == novaSenha) {
                Console.WriteLine("Acesso Permitido");
                break;
            } else {
                Console.WriteLine("Senha Invalida");
            }
        }
    }
}